addons/towing/functions/fnc_towStateMachinePFH.sqf

```diff
@@ -140,7 +140,7 @@ switch (_state) do {
         [_unit, _ropeClass, true] call CBA_fnc_addItem;
         _args set [0, TOW_STATE_CLEANUP];
         GVAR(cancel) = false;
-
+        [QGVAR(ropeDeployCanceled), [_unit, _ropeClass]] call CBA_fnc_localEvent;
```

Comment:
While both are technically correct `Cancelled` sounds more proper to me (British English)

cc @PabstMirror

Reply:
Nevermind sounding better, it looks better. IMO this is 1 where a rule should be established to decrease uncertainty and time wasted on bugs and looking for the used spelling; admittedly I dunno if the already inconsistent English language has such consistency in this kinda spelling difference.

Reply:
matches existing event names
https://github.com/acemod/ACE3/blob/master/addons/fortify/XEH_postInit.sqf#L97
but either is fine

Reply:
https://community.bistudio.com/wiki/Arma_3:_Event_Handlers#LandingCanceled

Reply:
Haven't thought about existing events. American English it is.